api/src/pcapi/core/offers/api.py

```diff
@@ -593,7 +593,10 @@ def create_stock(
                 stock=created_stock,
             )
         )
-    repository.add_to_session(created_stock, *created_activation_codes)
+    if offer.lastValidationPrice is None:
+        offer.lastValidationPrice = price
```

Comment:
Pourquoi remplir `lastValidationPrice` à la création du stock, et pas seulement au moment de la validation ?
Selon moi, c'est une info liée à ce qui a été validé.

Reply:
Parce qu'on peut valider une offre sans qu'elle n'ait de stock (cf l'API publique); dans ce cas, il faut remplir l'info à la création du premier stock

Reply:
Dans ce cas, on veut le faire uniquement si l'offre est déjà validée, non ?
Ce qui donnerait :
```suggestion
    if offer.isApproved and offer.lastValidationPrice is None:
        offer.lastValidationPrice = price
```

Reply:
En effet